ci_framework/roles/openshift_setup/README.md

```diff
@@ -0,0 +1,11 @@
+## Role: openshift_setup
+Prepares the target OpenShift cluster to be used by further steps:
+- Project/Namespace creation
+
+### Privilege escalation
+No privilege escalation needed.
+
+### Parameters
+* `cifmw_openshift_setup_kubeconfig`: (String) Path to the kubeconfig file. Defaults to `cifmw_kubeconfig`.
```

Comment:
`cifmw_openshift_setup_kubeconfig` is missing from default role vars. We can add or comment it out there.

Reply:
you can check how we reference "non-default variables" here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/repo_setup/defaults/main.yml#L34-L37